app/src/main/java/de/tum/in/tumcampusapp/component/ui/overview/MainActivity.java

```diff
@@ -106,7 +106,8 @@ protected void onCreate(Bundle savedInstanceState) {
         Intent service = new Intent(this, SilenceService.class);
         this.startService(service);
 
-        downloadNewsAlert();
+        // redundant call; DownloadService was already called by the StartupActivity
+        //downloadNewsAlert();
```

Comment:
We generally do *not* want to have commented-out code.
I think the documenting comment would rather fit the commit message.

So please remove it and I'll happily merge your PR.